docs-src/docs/setup/install.md

```diff
@@ -35,59 +37,93 @@ This step will create an Identity & Access Management (IAM) access key from your
 
 8. Select the group **Administrators** and click **Add to Groups**. This will make your newly created user an administrator for your AWS account, so be sure to keep your security credentials safe!
 
-## Part 3: Configure your Layer0
-Now that you have downloaded Layer0 and configured your AWS instance, you can create your Layer0.
+## Part 3: Create a new Layer0 Instance
+Now that you have downloaded Layer0 and configured your AWS account, you can create your Layer0 instance.
+Fom a command prompt, run the following (replacing `<instance_name>` with a name for your Layer0 instance:
+```
+$ l0-setup init <instance_name>
+```
 
-**To configure Layer0:**
+This command will prompt you for many different inputs. 
+Simply press `enter` on all of the optional inputs to use the default values.
```

Comment:
Slightly confusing sentence order here --- suggests that AWS access keys, etc may have default values. 

would prefer something like:
Enter the required values for **AWS Access Key**, **AWS Secret Key**, and **AWS SSH Key** as they come up. All remaining inputs are optional and can be set to their default by pressing enter.